bcda/api.go

```diff
@@ -55,7 +56,11 @@ import (
 )
 
 /*
+<<<<<<< HEAD
```

Comment:
I think a merge conflict messed this up right here.

Reply:
Thanks for catching that!